### PR TITLE
initial commit for test-feature-1

### PR DESCRIPTION
**Pull request by nguthrie**

### CODE REVIEW

File: test-feature-1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+code for new test feature (1)
```
Comment (neuromuted) [inline test-feature-1.txt]: this is not the feature I want! do better next time.

**Review by neuromuted — CHANGES_REQUESTED** [review]
you're bad

**Comment by nguthrie** [comment]
ouch, that hurts.

**Review by neuromuted — APPROVED** [review]
wow, so much better, so quickly!

**Comment by neuromuted** [comment]
new-feature-2 is good

**Comment by neuromuted** [comment]
new-feature-1 is good

**Pull request merged**